docs/real_world_fl/cloud_deployment.rst

```diff
@@ -0,0 +1,299 @@
+
+There are two ways to provision and deploy NVFLARE. The first way is to use the :ref:`NVFLARE CLI tool for provisioning <provision_command>`. However,
+the startup kits for different FL clients will need to be distributed manually via email, sftp, etc.
+
+The second way is to use the :ref:`NVFLARE Dashboard Web UI <dashboard_api>`, which allows project administrators to set up the FL project once
+participants are invited. Users can download the startup kit directly from the FLARE Dashboard without the need for manual distribution.
+
+Regardless of the approach, the user will essentially have a startup kit before trying to deploy the NVFLARE application. To deploy the NVFLARE FL
+system to the cloud. You just need the following commands (while substituting in the appropriate values):
+
+.. code-block:: shell
+
+    FL Server:    <startup dir>/start.sh     –cloud <csp>  [ –config config.json ]
+    FL Client:      <startup dir>/start.sh     –cloud <csp>  [ –config config.json ]
+    Dashboard:   <startup dir>/start.sh     –cloud <csp>  [ –config config.json ]
+
+
+Deploy NVFLARE Dashboard in the Cloud
+=====================================
+If you decide to use leverage NVFLARE Dashboard to provision and distribute your startup kit, you will first need to set up NVFLARE Dashboard.
+Let us assume that you are the project admin and would like to set up the NVFLARE Dashboard Web Application on the cloud, and you have all the
+needed credentials to access and create the cloud infrastructure. In Azure, your role in Azure
+subscription must be able to create Resource Group, Virtual Machine and configure Network Security Group and its rules.
+In AWS, your role should have AmazonEC2FullAccess.
+
+Create Dashboard on Azure
+-------------------------
+To run NVFlare dashboard on Azure, run:
+
+.. code-block:: shell
+
+    nvflare dashboard --cloud azure
+
+
+.. note::
+
+    The script also requires sshpass and jq.  Both can be installed on Ubuntu, with:
+
+        .. code-block:: shell
+
+           sudo apt install sshpass jq
+
+Users only need to enter an email address and press Enter. This user needs to remember this email and the temporary password that will be provided, as
+this is the login credentials for the NVFLARE Dashboard once the Dashboard is up and running. 
+
+Before you can create the infrastructure, Azure requires users to login with their credentials. This is handled by the official Azure CLI tool that you
+can install by following this page: https://learn.microsoft.com/en-us/cli/azure/install-azure-cli. The az login process will show a message similar to the following:
+
+.. code-block:: text
+
+    To sign in, use a web browser to open the page https://microsoft.com/devicelogin and enter the code A12GCJZR2 to authenticate.
+
+Please open the above URL and enter the random code (A12GCJZR2 in the above example). Then follow the entire login process, which will require user credentials and MFA.
```

Comment:
I did not add it for now because it contained the Microsoft logo and assets, and furthermore could potentially trigger some sort of phishing filter against our pages because it could be mistakenly categorized as a resource to fake a login page to attempt to gain someone's login credentials.